### PR TITLE
Work Page

### PR DESCRIPTION
**Pull request by IfzaRasool**
Is a recent post page where you upload your post and describe your page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+               <div class="tag">
+                <p>css</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>html</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>bootstrap</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>Ruby</p>
+                </div>
+                <div class="button"><img src="Icons/seeproject.png" alt="button"></div>
```
Comment (andersonlebon) [inline index.html:114]: - [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 
```
 <ul class="tools">
      <li>css </li>
      <li>html</li>
      <li>bootstrap</li>
      <li>Ruby</li>
</ul>

```

File: index.html [hunk 2]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+               <div class="tag">
+                <p>css</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>html</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>bootstrap</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>Ruby</p>
+                </div>
```
Comment (andersonlebon) [inline index.html:128]: ## The same thing here

- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 
```
 <ul class="tools">
      <li>css </li>
      <li>html</li>
      <li>bootstrap</li>
      <li>Ruby</li>
</ul>

```

File: index.html [hunk 3]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
```
Comment (andersonlebon) [inline index.html:143]: ## The same thing here

- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 
```
 <ul class="tools">
      <li>css </li>
      <li>html</li>
      <li>bootstrap</li>
      <li>Ruby</li>
</ul>

```

File: index.html [hunk 4]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
```
Comment (andersonlebon) [inline index.html:158]: ## The same thing here

- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 
```
 <ul class="tools">
      <li>css </li>
      <li>html</li>
      <li>bootstrap</li>
      <li>Ruby</li>
</ul>

```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @IfzaRasool 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+         
+        <div class="arrow-line">
+            <img src="Icons/arrow-bar.png" alt="arrow line">
+        </div>
+        <div class="arrow-head">
+            <img src="Icons/arrow-head.png" alt="arrow head">
+        </div>
+        <h2 class="work-text">Work</h2>
+
+        <section class="main-work-section">
```
Comment (IfzaRasool) [inline index.html:93]: > Use div instead section

File: index.html [hunk 6]
```diff
@@ -75,6 +76,95 @@ <h2 class="secondry-content">Software Developer</h2>
...
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+               <div class="tag">
+                <p>css</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>html</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>bootstrap</p>
+                <img src="Icons/Rectangle.png" alt="divide line">
+                <p>Ruby</p>
+                </div>
```
Comment (IfzaRasool) [inline index.html:113]: ddsss

**Review by IfzaRasool — COMMENTED** [review]
optional changes

**Pull request merged**